src/main/resources/assets/magicbees/lang/en_US.lang

```diff
@@ -38,6 +38,7 @@ item.beeRing.name=Collector's Bee Ring
 tile.effectJar.name=Bee Collector's Jar
 tile.magicApiary.name=Magic Apiary
 tile.visAuraProvider.name=Vis Apiary Booster
+tile.phialingCabinet.name=Phialing Cabinet
```

Comment:
Name needs to be updated, since the phialing cabinet model has been moved away from in this case, the class names probably need to be refactored as well.